src/Tribe/Field.php

```diff
@@ -340,6 +341,20 @@ public function doFieldName( $multi = false ) {
 		}
 
 		/**
+		 * returns the field's placeholder
+		 *
+		 * @return string the field value
+		 */
+		public function doFieldPlaceholder() {
```

Comment:
I prefer snake_case. While there may be merit in matching the neighbors for consistency, I _do_ like the idea of moving forward to what we actually want. Since this is for the current release, let's make this leap:

1. Use snake_case for this new method
2. Deprecate all the other camelCase methods and convert them to snake_case.